HospitalToday/Services/Implementation/DoctorService.cs

```diff
@@ -27,33 +29,40 @@ public void Remove(Person item)
             personRep.Delete(item.Id);
         }
 
-        public List<Person> GetList()
+        public IList<Person> GetList()
         {
             var persons = personRep.GetList().Where(x => x is Doctor);
```

Comment:
I think, that it will be better to merge this lines into something like this:
`personRep.GetList().Where(x => x is Doctor).ToList();`

Reply:
Done